app/components/Student.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+export default function Student({selectedStudent}) {
+   console.log('STUDENT COMPONENT : selectedStudent: ', selectedStudent);
+  const {firstName, lastName, email, campusId, campus} = selectedStudent;
+    console.log('campus: ', campus);
+
+  return (
+    <div>
+      <span>NAME: {`${firstName} ${lastName}`}<br/>
```

Comment:
The extra level of interpolation here isn't needed. The JSX syntax can handle this case.


```js
<span>NAME: {`${firstName} ${lastName}`}<br/>
```

is equivalent to:

```js
<span>NAME: {firstName} {lastName}<br/>
```